internal/services/appconfiguration/app_configuration_resource.go

```diff
@@ -132,6 +133,37 @@ func resourceAppConfiguration() *pluginsdk.Resource {
 				ValidateFunc: validation.StringInSlice(configurationstores.PossibleValuesForPublicNetworkAccess(), true),
 			},
 
+			"replica": {
+				Type:     pluginsdk.TypeSet,
+				Optional: true,
+				MinItems: 1,
+				Set:      resourceConfigurationStoreReplicaHash,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"name": {
+							Type:         pluginsdk.TypeString,
+							Required:     true,
+							ValidateFunc: validate.ConfigurationStoreReplicaName,
+						},
+						"location": {
```

Comment:
This needs to be added to the hash code since this can be updated. Also this should be using the location schema from the commonschema packages. There's a location schema without ForceNew.

Reply:
fixed, thanks.